src/foam/box/RetryBox.js

```diff
@@ -103,7 +103,7 @@ foam.CLASS({
     {
       name: 'maxAttempts',
       documentation: 'Set to -1 to infinitely retry.',
-      value: 3
+      value: 0
```

Comment:
Would be better to remove RetryBox from ClientBuilder.js for the time being, and leave a comment there explaining why, because we want to turn this back on when our Exceptions from the server are better categorized.

Reply:
or even retry: true/false to EasyDAO would work.